GetIntoTeachingApiTests/Models/Crm/Validators/CandidateValidatorTests.cs

```diff
@@ -106,32 +100,34 @@ public void Validate_WhenValid_HasNoErrors()
                 MobileTelephone = "07584 734 576",
                 HasDbsCertificate = true,
-                HasGcseMathsId = mockPickListItem.Id,
-                HasGcseEnglishId = mockPickListItem.Id,
-                AdviserEligibilityId = mockPickListItem.Id,
-                PlanningToRetakeGcseScienceId = mockPickListItem.Id,
-                PlanningToRetakeGcseEnglishId = mockPickListItem.Id,
-                TypeId = mockPickListItem.Id,
-                AssignmentStatusId = mockPickListItem.Id,
+                HasGcseMathsId = fakeWebsitehasgcseenglish.Id, //ToDo: Check this validation
+                HasGcseEnglishId = fakeWebsitehasgcseenglish.Id,
+                AdviserEligibilityId = fakeIscandidateeligibleforadviser.Id,
+                PlanningToRetakeGcseScienceId = fakeWebsiteplanningretakeenglishgcse.Id,
+                PlanningToRetakeGcseEnglishId = fakeWebsiteplanningretakeenglishgcse.Id,
+                TypeId = fakeTypeofcandidate.Id,
+                AssignmentStatusId = fakeTypeofcandidate.Id,
                 DoNotPostalMail = false,
                 EligibilityRulesPassed = "true",
-                ConsiderationJourneyStageId = mockPickListItem.Id,
+                ConsiderationJourneyStageId = fakeWebsitewhereinconsiderationjourney.Id,
                 CountryId = mockCountry.Id,
                 PreferredTeachingSubjectId = mockSubject.Id,
-                PreferredEducationPhaseId = mockPickListItem.Id,
-                InitialTeacherTrainingYearId = mockPickListItem.Id,
-                ChannelId = mockPickListItem.Id,
-                MailingListSubscriptionChannelId = mockPickListItem.Id,
-                EventsSubscriptionChannelId = mockPickListItem.Id,
-                TeacherTrainingAdviserSubscriptionChannelId = mockPickListItem.Id,
-                ApplyPhaseId = mockPickListItem.Id,
-                ApplyStatusId = mockPickListItem.Id,
-                Situation = mockPickListItem.Id,
-                PrivacyPolicy = new CandidatePrivacyPolicy() { AcceptedPolicyId = (Guid)mockPrivacyPolicy.Id }
+                PreferredEducationPhaseId = fakePeferrededucationphase.Id,
+                InitialTeacherTrainingYearId = fakeIttyear.Id,
+                ChannelId = fakeChannelcreation.Id,
+                MailingListSubscriptionChannelId = fakeGitismlservicesubscriptionchannel.Id,
+                EventsSubscriptionChannelId = fakeGitiseventsservicesubscriptionchannel.Id,
+                TeacherTrainingAdviserSubscriptionChannelId = fakeGitisttaservicesubscriptionchannel.Id,
+                ApplyPhaseId = fakeCandidateapplyphase.Id,
+                ApplyStatusId = fakeCandidateapplystatus.Id,
+                Situation = fakeSituation.Id,
+                PrivacyPolicy = new CandidatePrivacyPolicy() { AcceptedPolicyId = (Guid)mockPrivacyPolicy.Id },
+                Citizenship = fakeCitizenship.Id,
+                VisaStatus = fakeVisastatus.Id
```

Comment:
add Location

consider moving Situation down to this section to keep these 4 fields together